src/components/PageError.jsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from "react-router-dom";
+
+export default function PageError({ clientMessage, error = false }) {
+	console.log(error);
```

Comment:
pls remove log

Reply:
darnit - I do my best looking for these but I miss my spyon console test in Jest! Cleaned up now.